src/bin/exrinfo/CMakeLists.txt

```diff
@@ -2,10 +2,17 @@
 # Copyright Contributors to the OpenEXR Project.
 
 add_executable(exrinfo main.c)
-target_link_libraries(exrinfo OpenEXR::OpenEXRCore)
+target_link_libraries(exrinfo OpenEXR::OpenEXRCore OpenEXR::Config)
 set_target_properties(exrinfo PROPERTIES
   RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
 )
+
```

Comment:
I think this is wrong, and the problem lies with OpenEXRCore. A program as simple as `exrinfo` shouldn't require anything more than stating OpenEXRCore as a link library.

If you add `DEPENDENCIES Imath::Imath` to `src/lib/OpenEXRCore/CMakeLists.txt`, it should all work.

Reply:
Ok, letting the CI have a go with that idea!